File: tests/fixtures/anchor/sample.py
class Foo:
    def bar(self, a):
        if a:
            for x in [1]:
                return x
        return 0

    @staticmethod
    def baz():
        return list(map(lambda v: v, map(lambda v: v + 1, [1])))

def top():
    return 1

File: tests/fixtures/anchor/sample.php
<?php
namespace App;
class Foo {
    public function bar(int $a): int { if ($a) { foreach ([1] as $x) { return $x; } } return 0; }
    public function baz(): array { return array_map(function ($v) { return $v; }, array_map(fn($v) => $v, [1])); }
}
function top(): int { return 1; }
